### DISCUSSION TITLE
How to best get statistics for usage?

### DISCUSSION THREAD

**Discussion by dpriskorn** [LeMyst/WikibaseIntegrator#260] [Ideas]
https://github.com/LeMyst/WikibaseIntegrator/issues/181#issuecomment-903253080
What about a new repo where people using the script can voluntarily write a pull request with a link to their code and the user-agent they use?

**Comment by LeMyst** [comment]
I'm afraid that ApiFeatureUsage is not reliable enough. I've random statistics when I try to query the api.
But a repository with both wikidata/wikibase bot and tools curated list will be good.
